src/main/java/nl/avans/drivio/service/CarService.java

```diff
@@ -1,7 +1,7 @@
 package nl.avans.drivio.service;
 

```

Comment:
Dit hele bestand moet verwijderd worden om in lijn te zijn met de code van de klassen car

Reply:
Fixed met merge conflicts in commit https://github.com/leonvsc/Drivio/pull/26/commits/d6c224c192476645187e85243bfa37e53c615ef4